### ISSUE TITLE
Rename Audio class to XAudio2Engine

### ISSUE THREAD

**Issue by MFHSchoonbrood** [cocos2d-x/cocos2dx-win8#24]
Could you rename the Audio class used by the SimpleAudioEngine to XAudio2Engine?
As the very generic Audio name is bound to conflict with other generic audio classes.

I already did it locally and can provide you with the code (renamed Audio class and updated SimpleAudioEngine class), if you're interested.

Kind regards,
Michaël

**Comment by saudahmed** [comment]
Michaël were you able to play a soundfile using SimpleAudioEngine for this Branch

**Comment by MFHSchoonbrood** [comment]
Yes, I have my whole game working on Windows 8 tablet now (tested on ASUS vivotab).

**Comment by saudahmed** [comment]
Michaël can you share a code snippet for the fix that you induced in your game for it. It would be nice of you

**Comment by saudahmed** [comment]
Michaël this issue is for 2.1 Api

**Comment by MFHSchoonbrood** [comment]
My issue was for the cocos2dx-win8 code (master branch), not the Cocos2d-x for wp8 v2.0 preview (wp8_v2 branch).
Are you talking about the later? As I haven't looked at it because it seems to be for Windows Phone, instead of Windows 8 tablets.

I only now notice that they are on the same github, after you posted this :)

EDIT: I looked at the WP8 V2.0 code as well, and the audio code is mostly the same (same issue on "Audio" class name), with the big difference being that is uses the mediastreamer to load the sound. I'm assuming that this is related to ARM code. I haven't tested it out in my game, just compared the code to the MASTER branch.
